cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -630,14 +630,6 @@
   </property>
 
   <property>
-    <name>explore.executor.container.instances</name>
```

Comment:
Please check with @awholegunch as well. I think this will break the doc build. You'll have to deprecate this property first before removing in the next release.